DAI-Zenon.js

```diff
@@ -84,54 +81,43 @@ function doWeb(doc, url) {
 }
 
 function scrape(doc, url) {
-	//e.g. url = "http://zenon.dainst.org/#book/000300287"
+	
+	//e.g. url = "http://zenon.dainst.org/Record/000300287"
 	var urlParts = url.split("/");
 	var id = urlParts[urlParts.length-1];
-	var jsonUrl = "http://zenon.dainst.org:8080/elwms-zenon/resource/" + encodeURIComponent(id) + "?format=marc21";
 	
-	ZU.doGet(jsonUrl, function(text) {
-		var json = JSON.parse( text );
-		var rootElement = "data";
-		var fields = json[rootElement]["marc:datafield"];
-		//Z.debug(fields);
-
-		//call MARC translator
+	//call MARC translator
+	ZU.doPost('http://zenon.dainst.org/Record/' + id + '/AjaxTab', 'tab=details', function(text) {
```

Comment:
It is no XML but the MARC codes are embedded in the table structure of the HTML site:

![dai-marc](https://cloud.githubusercontent.com/assets/5199995/18463331/986ba9c6-7989-11e6-9a79-6bc6ed899af7.png)